packages/astro/src/@types/astro.ts

```diff
@@ -67,12 +67,19 @@ export interface JsxItem {
   jsx: string;
 }
 
+export interface ScriptInfo {
+  src?: string;
+  content?: string;
+}
+
 export interface TransformResult {
   script: string;
   imports: string[];
   exports: string[];
+  components: string[];
   html: string;
   css?: string;
+  scripts: ScriptInfo[];
```

Comment:
`script` and `scripts` feels confusing. `inlineScripts` since you have `hoistedScripts` below?

Reply:
I'll change to `hoistedScripts` here too, inline wouldn't be right because they could be inline or external.